src/collective/sitter/browser/configure.zcml

```diff
@@ -61,12 +61,14 @@
       allowed_attributes="breadcrumbs"
       />
 
+  <adapter factory=".sitter.SitterContactAdapter" />
+
   <browser:page
     name="sitterview"
     for="..content.sitter.ISitter"
-    class=".sitter.SitterView"
+    layer="zope.interface.Interface"
```

Comment:
Leave out layer altogether?

Reply:
`layer=".layer.ICollectiveSitterLayer"` should be better. But it is not working and I have no idea why.

Reply:
It is working for me, unless I tried something different than you did. Maybe the profile you used for testing hadn't been reinstalled for long enough so the layer wasn't activated yet?

Reply:
Now it works for me too. No idea why. So I have added `layer=".layer.ICollectiveSitterLayer"`